win32/signal.c

```diff
@@ -129,7 +127,7 @@ PHP_FUNCTION(sapi_windows_set_ctrl_handler)
 	}
 
 	zval_dtor(&ctrl_handler);
-	ZVAL_COPY(&ctrl_handler, handler);
+	ZVAL_COPY(&ctrl_handler, &fci.function_name);
```

Comment:
That won't work, since `ctrl_handler` is a static global, which is declared as `zval` and used elsewhere in this file as well.

Reply:
Nonsense. Sorry for the noise.

Reply:
The noise did make me double check and realize I didn't get rid of the ``handler`` variable which was used in between.